interfaces/builtin/u2f_devices.go

```diff
@@ -147,6 +147,16 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "32a3",
 		ProductIDPattern: "3201",
 	},
+	{
+		Name:             "Trezor",
+		VendorIDPattern:  "534c",
+		ProductIDPattern: "0001",
```

Comment:
https://devicehunt.com/search/type/usb/vendor/534C/device/any reports 0002 for a v2 version. It probably makes sense to add it as well.